sdk-springboot/src/main/java/io/dapr/springboot/DaprRuntime.java

```diff
@@ -36,9 +34,10 @@ class DaprRuntime {
   private final Map<DaprTopicKey, DaprSubscriptionBuilder> subscriptionBuilders = new HashMap<>();
 
   /**
-   * Private constructor to make this singleton.
+   * DaprRuntime should be used as a singleton, using {@link DaprRuntime#getInstance()}.
+   * The constructor's default scope is available for unit tests only.
    */
-  private DaprRuntime() {
+  DaprRuntime() {
```

Comment:
Is this necessary ? If not can we keep it private only ?

Reply:
Please, use reflection in the unit test to instantiate this class and keep it private. This is important, otherwise users can mess this up.

This blog post describes how to do it: https://avaldes.com/how-do-i-instantiate-a-class-with-a-private-constructor/

Reply:
Since the scope is `default`, wouldn't it disallow users from accessing it? For now I have reverted it back to private and used reflection in test.